RedHat-UX/red-hat-design-system
# fix(tag): outline variant

## Related issues
#987 
#934 

## What I did

1.  Added a fix into `rh-tag` to add the color `white` and the `outline` variant styles.
2. Added the content into the docs pages into `rh-tag` from #934

## Testing Instructions

1. Please go to the DP.
2. Check the following tabs under `rh-tag` and ensure the content matches:
  - `Overview`
  - `Style`
  - `Guidelines`
  - `Code`
  - `Accessibility`
